tests/e2e/actions.go

```diff
@@ -599,45 +872,29 @@ func (tr Chain) submitConsumerModificationProposal(
 	action SubmitConsumerModificationProposalAction,
 	verbose bool,
 ) {
-	template := `
 
-{
-"messages": [
-  {
-    "@type": "/interchain_security.ccv.provider.v1.MsgConsumerModification",
-	"title": "Propose the modification of the PSS parameters of a chain",
-	"description": "description of the consumer modification proposal",
-	"chain_id": "%s",
-	"top_N": %d,
-	"validators_power_cap": %d,
-	"validator_set_cap": %d,
-	"allowlist": %s,
-	"denylist": %s,
-    "authority": "cosmos10d07y265gmmuvt4z0w9aw880jnsr700j6zn9kn",
-    "min_stake": %d,
-    "allow_inactive_vals": %t
-  }
- ],
-"metadata": "ipfs://CID",
-"deposit": "%sstake",
-"title": "Propose the modification of the PSS parameters of a chain",
-"summary": "summary of a modification proposal",
-"expedited": false
- }
-`
-
-	jsonStr := fmt.Sprintf(template,
-		string(tr.testConfig.chainConfigs[action.ConsumerChain].ChainId),
-		action.TopN,
-		action.ValidatorsPowerCap,
-		action.ValidatorSetCap,
-		action.Allowlist,
-		action.Denylist,
-		action.Deposit,
-		action.MinStake,
-		action.AllowInactiveVals,
-	)
+	consumerId := string(tr.testConfig.chainConfigs[action.ConsumerChain].ConsumerId)
+	title := "Propose the modification of the PSS parameters of a chain"
+	description := "description of the consumer modification proposal"
+	summary := "summary of a modification proposal"
+	expedited := false
+	metadata := "ipfs://CID"
+	deposit := fmt.Sprintf("%dstake", action.Deposit)
+	authority := "cosmos10d07y265gmmuvt4z0w9aw880jnsr700j6zn9kn"
+
+	msgConsMod := types.MsgUpdateConsumer{
```

Comment:
What does `ConsMod` refer to? Why not `msgUpdateConsumer` in similar fashion with previous `msgRemoveConsumer`?